.jshintignore

```diff
@@ -1,2 +1,3 @@
 app/bower_components
-node_modules
\ No newline at end of file
+node_modules
+public
```

Comment:
This lets you run `$ jshint .` from the Terminal without getting a bunch of bogus errors in "public/" (although you should really be using the `grunt jshint` task anyways).